src/background/direct-link-query.js

```diff
@@ -4,7 +4,7 @@
  *
  * See https://h-client.readthedocs.io/en/latest/publishers/config/#config-settings
  *
- * @typedef {Object} Query
+ * @typedef {object} Query
```

Comment:
Current conventions for typedefs are to omit the `{object}` part if there are `@prop` definitions below, because it can be inferred in that case.

Reply:
👍🏼